src/sys/windows.rs

```diff
@@ -92,29 +90,31 @@ pub fn reflink(from: &Path, to: &Path) -> io::Result<()> {
             debug_assert_eq!(bytes_copied % cluster_size, 0);
         }
         unsafe {
```

Comment:
This is about a line above, not this one.

Can you change `cmp::min(a,b)` to `a.min(b)`? That's just weird

Reply:
Done!